src/main/scala/com/jcdecaux/setl/storage/connector/CassandraConnector.scala

```diff
@@ -166,4 +169,26 @@ class CassandraConnector(val keyspace: String,
       .where(query)
       .deleteFromCassandra(keyspace, table)
   }
+
+  override def drop(): Unit = {
+    cqlConnection.withSessionDo {
+      session =>
+        session.execute(s"DROP TABLE IF EXISTS $keyspace.$table;")
+    }
+  }
+
+  def createKeyspace(strategy: String, replicationFactor: Int): Unit = {
+    cqlConnection.withSessionDo {
+      session =>
+        session.execute(s"CREATE KEYSPACE IF NOT EXISTS $keyspace WITH replication = {'class':'$strategy', 'replication_factor':$replicationFactor};")
```

Comment:
Should we check if strategy is ```SimpleStrategy``` or ```NetworkTopologyStrategy``` and throw an error if it is not one of them ?

Reply:
maybe we should let spark-cassandra connector to handle the exception. i will declare the exception in the method